index.html

```diff
@@ -771,17 +778,14 @@ <h2 id="intro">Introduction</h2>
         <p>There is currently no reasonable, scalable, generalizable and sustainable method to provide information about mathematical rendering using HTML with CSS or SVG to screen readers and braille displays.</p>
 
         <p>While MathML is part of HTML5, virtually all mathematical and scientific content on the web is rendered using HTML (with CSS) and SVG. While these standards are sufficient for visual rendering, there is no consistent method for content creators to make mathematical and scientific information available to a non-visual display such as a screen reader.</p>
-        <p><strong>Suggestion: </strong>A joint Task Force of the W3C's ARIA WG and DPub IG could investigate potential solutions such as a mathematics/STEM-related ARIA module.</p>
 
+        <p>The newly created [[Getting Math Onto WebPages]] Community Group plans to address rendering issues as well as accessibility related issues, focusing on a bottom-up approach of improving the web platform overall to help improve current and future technologies.</p>
 
-		<p>Contact:</p>
-		<ul>
-		   	<li>None at this time.</li>
-		</ul>
+        <p><strong>Suggestion: </strong>The DPub IG could follow and support the efforts of the math-on-webpages Community Group and encourage a joint Task Force between the Community Group and the W3C's ARIA WG to investigate potential solutions such as a mathematics/STEM-related ARIA module.</p>
 
-        <p>References:</p>
+    	<p>Contact:</p>
         <ul>
-             <li>[[IDPF-Guide-Logical-Reading-Order]]</li>
```

Comment:
Since it came up by private email, this reference was a copy&paste error in the first version of this section.